astronomer/providers/databricks/example_dags/example_databricks.py

```diff
@@ -13,8 +13,7 @@
 
 DATABRICKS_CONN_ID = os.environ.get("ASTRO_DATABRICKS_CONN_ID", "databricks_default")
 # Notebook path as a Json object
-# Example: {"notebook_path": "/Users/pankaj.singh@astronomer.io/quick_start"}
-notebook_task = '{"notebook_path": "/Users/pankaj.singh@astronomer.io/quick_start"}'
+notebook_task = '{"notebook_path": "/Users/phani.kumar@astronomer.io/quick_start"}'
```

Comment:
How can I get to see this notebook?

Reply:
This should be available in databricks UI. you can login to databricks from okta